identity/aziot-identity-common/src/lib.rs

```diff
@@ -31,6 +32,13 @@ pub struct AzureIoTSpec {
     pub auth: Option<AuthenticationInfo>,
 }
 
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct LocalIdSpec {
```

Comment:
missing module_id field. Also, the auth info should be nested for extensibility (since you were looking at supporting other forms of auth?)

Reply:
Fixed